docroot/sites/all/themes/custom/wsdot_classic/css/styles_orange.css

```diff
@@ -1,407 +1,10 @@
-
-.buyTicket_button{float: right;width:250px; height:26px;}
-.buyTicket_b{
-	display: block;
-	width:250px;
-	height:26px;
-	background-color: #eea537;
-	color: white;
-	text-decoration: none;
-	text-align: center;
-	font-family: "Myriad Pro","Segoe UI",Helvetica, Arial, sans-serif;
-	font-size: 14px;
-	font-weight: bold;
-	text-transform: uppercase;
-	padding-top: 6px;
-}
-.buyTicket_b:visited{
-	background-color: #eea537;
-	color: white;
-	text-decoration: none;
-	text-align: center;
-	font-family: "Myriad Pro","Segoe UI",Helvetica, Arial, sans-serif;
-	font-size: 14px;
-	font-weight: bold;
-	text-transform: uppercase;
-	font-weight: bold;
-	padding-top: 6px;
-}
-.buyTicket_b:hover{
-	background-color: #f5d134;
-	color: white;
-	text-decoration: none;
-	text-align: center;
-	font-family: "Myriad Pro","Segoe UI",Helvetica, Arial, sans-serif;
-	font-size: 14px;
-	font-weight: bold;
-	text-transform: uppercase;
-	font-weight: bold;
-	padding-top: 6px;
-}
-
-/*====Ferries Reservation====*/
-.heading{height:28px;	background:url(http://www.wsdot.wa.gov/media/images/ferries/green_bkg.jpg); padding: 10px 0 0 0; margin: 0 0 10px 0;  }
-/*============== mod =============*/
-
-.button_sm {display: inline-block;	width: 130px;cursor: pointer;text-align: center;text-decoration: none;margin:10px; padding:5px;-webkit-border-radius: 10px;-moz-border-radius:10px;border-radius: 10px; color: #fff;border: solid 1px #01513f;background: url(http://www.wsdot.wa.gov/media/images/ferries/green_bkg.jpg)}
-.button_sm:hover:{width:130px; background: url(http://www.wsdot.wa.gov/media/images/ferries/green_bkg_hover.jpg);border: solid 1px #019573;}
-.green {color: #fff;border: solid 1px #01513f;background: url(http://www.wsdot.wa.gov/media/images/ferries/green_bkg.jpg)}
-h4.inverse{color:#FFF;}
-h4.inverse a{color:#fff;}
-p.inverse{color:#FFF; }
-p.inverse a{color:#fff;}
-table.ferriesres {border-collapse: collapse; border: 3px solid #ccc; margin:0 0 15px 0;}
-table.ferriesres td {border-collapse: collapse; border: 2px solid #ccc;}
-.ptbs{padding:10px;}
-.plm {padding-left:10px;}
+/* Orange */
+a {color:#cc3300;}
+a:visited {color:#6a1a00;}
+#header .topban {background-color: #ff6600;height:2.5em;}
+input.searchButton {color:#ffffff;font-family:'trebuchet ms',helvetica,sans-serif;font-size:1.1em;font-weight:bold;background-color:#ff6600;border:1px solid;border-top-color:#bdbdbd;border-left-color:#bdbdbd;border-right-color:#CC3333;border-bottom-color:#cc3300;}
+.leftnavbox .content a {color:#cc3300;}
+.leftnavbox .content a:visited {color:#6a1a00;}
+.logo h1 a {background: transparent url(http://www.wsdot.wa.gov/media/images/OrangeLogo.gif) no-repeat; display: block; width: 250px; height: 39px; text-indent: -999em;}
+.mainnav ul li a {background:transparent url(http://www.wsdot.wa.gov/media/images/back/tabs_link_bg.png) no-repeat right top;color:#ff6600;display:block;height:17px;line-height:1.5em;padding:7px 13px 6px 7px;text-decoration:none;float:left;}
```

Comment:
Looks good, however, could you update the background images to be relative to the Drupal images directory rather than the static HTML server. Thanks!